### ISSUE TITLE
Flatpak support

### ISSUE THREAD

**Issue by orowith2os** [Frogging-Family/wine-tkg-git#989]
Wine-tkg currently builds using the system (Arch, Fedora, etc) packages, however it might be beneficial to build it inside of Flatpak instead; this gives it a stable ABI to target. It can also run on the native system after building, but it would be somewhat similar to disabling the Steam Linux Runtime with Proton.

The layout would be like so:

- a Flatpak manifest would be bundled, which builds wine-tkg
- wine-tkg would then take the resulting binaries (which are in build-dir) and can package them into something usable by Steam and the like.
- building would be triggered either by running Flatpak-builder or a helper script.

**Comment by Fox2Code** [comment]
There is some problems with this approach, one of them is due to the sandboxed nature of flatpacks, so system integration is very limited and impossible in some cases. (binfmt config for example)

Also you can use Lutris instead if you want to run wine-tkg builds on a stable runtime.

However I can see the interest of having wine-tkg as a flatpack library for developers who want to port their games to Linux via flatpack.

**Comment by Fox2Code** [comment]
Yeah I misspelled flatpak cause I don't use them that much, but what you just said was mean!

![🦊](https://media.discordapp.net/stickers/860597321145909268.webp?size=160)

I just wanted to help and provide constructive criticism to your idea.

Like you said flatpak is a stable ABI, I think making wine-tkg able to be used as a library on flatpak would be a good thing.

But what you said was not nice to me.
